pythainlp/corpus/common.py

```diff
@@ -336,3 +337,26 @@ def thai_synonyms() -> dict:
 def thai_synonym() -> dict:
     warnings.warn("Deprecated: Use thai_synonyms() instead.", DeprecationWarning)
     return thai_synonyms()
+
+
+def find_synonyms(word) -> Union[List[str], None]:
+    """
+    Find synonyms
+
+    :param str word: Thai word
+    :return: List synonyms of word or None if it isn't exist.
+    :rtype: Union[List[str], None]
+
+    :Example:
+    ::
+
+        from pythainlp.corpus import find_synonyms
+
+        print(find_synonyms("หมู"))
+        # output: ['จรุก', 'วราห์', 'วราหะ', 'ศูกร', 'สุกร']
+    """
+    _temp = thai_synonyms()
```

Comment:
Name mangling (`_xxx`, or `__xxx` for classes) is a naming convention to indicate that the variable/function/method is meant to be private, shouldn't be accessible from outside the package/class.

For variables/functions inside function, they are already not accessible. So there's no need to put `_` prefix there.

I would suggest to change `_temp` and `_idx` to `temp` and `idx`. Or even better, `synonyms` and `idx` for clarity.

Reply:
Done